lib/testing/src/lib/process-services/pages/start-process.page.ts

```diff
@@ -145,9 +152,21 @@ export class StartProcessPage {
         return new FormFields();
     }
 
+    async checkSelectProcessField(): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsNotVisible(this.disabledSelectProcessDropdown);
```

Comment:
can we have a name like checkSelectProcessDropdownIsEnabled?

Reply:
Renamed